test/config_cat/cache_policy/auto_test.exs

```diff
@@ -0,0 +1,132 @@
+defmodule ConfigCat.CachePolicy.AutoTest do
```

Comment:
It's nice that with this split the tests are also now on different files, looks much cleaner.